src/eve-server/manufacturing/FactoryDB.cpp

```diff
@@ -257,7 +257,7 @@ PyRep *FactoryDB::AssemblyLinesSelectPublic(const uint32 regionID) {
     DBQueryResult res;
 
     if (!sDatabase.RunQuery(res,
-        "SELECT"
+        "SELECT DISTINCT"
```

Comment:
Without the DISTINCT I'd end up with 10 entries filter section on the Science & Industry tab if there were 10 slots.  There should only be 1 entry PER station/pos/outpost here and then upon selecting that entry it should show the 10 items below.